### DISCUSSION TITLE
settings.gradle.kts under build-logic module.

### DISCUSSION THREAD

**Discussion by momintahir** [android/nowinandroid#1466] [General]
HI, I was checking out this repository and I found out that there is a file named "settings.gradle.kts" under build-logic module. Please guide me where did it come from. Is it custom made ? Because if we create a module it doesn't come along with it.  
Note: I have also read the documentation regarding build-logic but I couldn't find any clue there as well.

**Comment by SimonMarquis** [comment]
Hi @momintahir, this is part of the regular setup of Gradle included builds.
You can read more on that in the Gradle documentation https://docs.gradle.org/current/userguide/composite_builds.html

**Reply by Jaehwa-Noh** [reply]
And, you can get the additional information what is the `settings.gradle` from this documentation.
https://docs.gradle.org/current/userguide/settings_file_basics.html

**Reply by momintahir** [reply]
Thanks a lot!
